docs/workflows/collections.rst

```diff
@@ -256,22 +237,53 @@ config (e.g. ``~/.ansible.cfg`` or ``/etc/ansible/ansible.cfg``):
 .. code::
 
     [galaxy]
-    server: http://localhost/pulp_ansible/galaxy/my_content/
+    server_list = pulp
```

Comment:
Is deprecated missing here on purpose?

Or asked the other way around: Can you still specify `-s deprecated` if it is not included here? Just reading the docs linked above didn't enlighten me about that.

Reply:
Yes. This demonstrates that the `deprecated` server won't be used by default, but you can still specify `-s dreprecated` if you need to download something from there. I'll clarify.

Reply:
Thank you. We will wind up with better docs than over there...